internal/services/iothub/iothub_endpoint_servicebus_queue_resource.go

```diff
@@ -119,6 +120,14 @@ func resourceIothubEndpointServicebusQueue() map[string]*pluginsdk.Schema {
 			ConflictsWith: []string{"identity_id"},
 			ExactlyOneOf:  []string{"endpoint_uri", "connection_string"},
 		},
+
+		// NOTE: O+C : required since this property would always be set even if it isn't specified in the tf config, otherwise it would cause a diff and break existing users
+		"subscription_id": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.IsUUID,
```

Comment:
```suggestion
		"subscription_id": {
			Type:         pluginsdk.TypeString,
			Optional:     true,
			// NOTE: O+C : required since this property would always be set even if it isn't specified in the tf config, otherwise it would cause a diff and break existing users
			Computed:     true,
			ValidateFunc: validation.IsUUID,
```

Reply:
Updated